docs/tutorial/04_writing-and-compiling-contracts.md

```diff
@@ -4,6 +4,7 @@ We're going to create a simple smart contract that implements a token that can b
 
 - There is a fixed total supply of tokens that can't be changed.
 - The entire supply is assigned to the address that deploys the contract.
+- The first user to ask for an initial stake receives a thousand tokens from the owner (this makes it easier to check the token software).
```

Comment:
Nice idea!

Reply:
Thank you. I'm lazy, and I figure my readers will also be lazy.